Page viewing permissions

HrabCOrp opened the pull request:
Changed: pages can be viewed only by their respective owners
Changed: redirections
  - site root redirects to login, if not logged in
  - site root redirects to the first page of the user, if logged in

  - login page redirects to the first page of the user, on successful log in
  - login page redirects to the first page of the user, if logged in

  - registration page redirects to login page on successful registration
  - registration page redirects to first page of the user, if logged in

rsviatoshchuk commented on plandarm/accounts/models.py:
```diff
@@ -14,10 +14,8 @@ def __str__(self):
 @receiver(post_save, sender=User)
 def create_user_profile(sender, instance, created, **kwargs):
     if created:
-        # this shit doesn't understand the GRAND SCHEME OF THINGS
```
.

rsviatoshchuk commented on plandarm/pages/urls.py:
```diff
@@ -1,7 +1,6 @@
-from django.urls import path
+from django.urls import path, include
```
?

rsviatoshchuk reviewed (changes requested):
Many blank line changes

HrabCOrp committed:
Page viewing permissions (fix)

Changed: pages can be viewed only by their respective owners
Changed: redirections
  - site root redirects to login, if not logged in
  - site root redirects to the first page of the user, if logged in

  - login page redirects to the first page of the user, on successful log in
  - login page redirects to the first page of the user, if logged in

  - registration page redirects to login page on successful registration
  - registration page redirects to first page of the user, if logged in

rsviatoshchuk reviewed (approved):
OK
